Sources/PerformanceTest/main.swift

```diff
@@ -44,6 +46,14 @@ router.get("json") { _, _ in
 
 var app = HBApplication(
     responder: router.buildResponder(),
+    channelSetup: try HTTP2Channel(
+        tlsConfiguration: .makeServerConfiguration(
+            certificateChain: [
+                .certificate(.init(file: "/Users/joannisorlandos/git/hummingbird/hummingbird/cert.pem", format: .pem))
```

Comment:
Path is only valid for yourself. It looks like we'll need to include AsyncHTTPClient to test HTTP2 fully. I think we should do that in a future PR though

Reply:
Yeah, I didn't need to commit this you're right. I've been using `curl` to test HTTP2 and HTTP1.1 negotiation + reading